### ISSUE TITLE
'bb export' removes tags from portal items

### ISSUE THREAD

**Issue by dmitrisweb** [Backbase/bb-cli#153]
While saving chunked portal, tags got removed from the portal items
`bb export --type portal --portal my-portal --save my-portal --chunk --force`

the problem is here: https://github.com/Backbase/bb-cli/blob/nightly/commands/export.js#L448
in the code it is expected that tag will be always an array, but if there is only one tag, it is not converted to the array:

Multiple tags:
```
{ tag:
   [ { '$blacklist': 'false',
       '$manageable': 'true',
       '$type': 'section',
       _: 'banking' },
     { '$blacklist': 'false',
       '$manageable': 'true',
       '$type': 'regular',
       _: 'my-tag' } ] }
```

Single tag:
```
{ tag:
   { '$blacklist': 'false',
     '$manageable': 'true',
     '$type': 'regular',
     _: 'web' } }
```

**Comment by dmitrisweb** [comment]
workaround would be: create more than one tag if you want them to be exported.

**Comment by santam85** [comment]
Closed by PR #154
